Say Their Names/Source/Controller/Home/HomeController.swift

```diff
@@ -135,6 +152,7 @@ extension HomeController {
 
         let completion = {
             self.revealTabBar()
+            self.homeView.hideable.forEach { $0.isHidden = false }
```

Comment:
this shouldn't be needed anymore after latest fixes to launch screen code - here too

Reply:
see note for above.